tests/components/laundrify/__init__.py

```diff
@@ -9,7 +9,7 @@
 from tests.common import MockConfigEntry
 
 
-def create_entry(
+async def setup_config_entry(
```

Comment:
Should this be a test fixture instead?

Reply:
Transformed into test fixture in https://github.com/home-assistant/core/pull/121378/commits/19a592be87a9938831357ab85d5870f676b06b12